inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -33,11 +33,7 @@ std::vector<std::string> disabledTestPatterns() {
         ".*DSR_GatherStaticDataDynamicIdx.*f32.*1.3.200.304.*",
         // TODO: Issue 47315
         ".*ProposalLayerTest.*",
-        // TODO: Issue 46755
-        ".*DSR_GatherElements.*",
-        // TODO: Issue 46756
-        ".*smoke_Gather_GatherElements.*",
         // TODO: Issue 48183
-        R"(.*CTCGreedyDecoderSeqLen.*?\(1.1.1\).*)",
+        R"(.*CTCGreedyDecoderSeqLen.*?\(0.1.1\).*)",
```

Comment:
@andrejsokolov, why did it changed? `1.1.1` topology hangs. Please, abort the test build and revert this change.

Reply:
Fixed. Thank you!